tools/mkssldef.py

```diff
@@ -33,6 +33,8 @@
       assert meta[2] in ('FUNCTION', 'VARIABLE')
       if meta[0] != 'EXIST': continue
       if meta[2] != 'FUNCTION': continue
+      # In cases where meta[3] is 'DEPRECATEDIN_X_Y,CATEGORY'
+      meta[3] = meta[3].split(',')[-1]
```

Comment:
Can you check that the zeroth field is what you expect it to be?
```python
if ',' in meta[3]:
  fst, meta[3] = meta[3].split(',')
  assert fst.startswith('DEPRECATEDIN')
```
The above assumes at most two fields. You could also filter out all deprecation tags:
```python
words = meta[3].split(',')
words = [word for word in words if not word.startswith('DEPRECATEDIN')]
assert len(words) == 1
meta[3] = words.pop()
```